<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pus0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pus3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusadd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusmin">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>180</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusmul">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusdiv">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>180</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusrem">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusroot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusfact">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="puspow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusback">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>˂</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pusclc">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>130</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="operator_2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>30</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="input2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="ans">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="res">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalculator">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionfa"/>
   </widget>
   <addaction name="menuCalculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionfa">
   <property name="text">
    <string>fa</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
